doc/rados/operations/monitoring.rst

```diff
@@ -345,19 +345,31 @@ the following::
 
 	ceph df
 
-The **RAW STORAGE** section of the output provides an overview of the
-amount of storage that is managed by your cluster.
+The output of ``ceph df`` for a theoretical 250 Kb object written to the 'foo_replicated' pool looks like this::
+
+        GLOBAL:
+        SIZE AVAIL USED RAW USED %RAW USED OBJECTS
+        30G 27.0G 10.3M 3.01G 10.03 22
+        POOLS:
+        NAME ID QUOTA OBJECTS QUOTA BYTES USED %USED MAX AVAIL OBJECTS DIRTY READ WRITE STORED USED COMPR UNDER COMPR
+        cephfs_data_a 1 N/A N/A 0 0 8.90G 0 0 0 0 0 0 0
+        cephfs_metadata_a 2 N/A N/A 1.50M 0 8.90G 21 21 0 42 2.19K 0 0
+        foo_replicated 4 N/A N/A 768K 0 8.90G 1 1 0 1 250K 0 0
+
+**GLOBAL** section now has USED and RAW USED columns. 
 
-- **CLASS:** The class of OSD device (or the total for the cluster)
 - **SIZE:** The amount of storage capacity managed by the cluster.
 - **AVAIL:** The amount of free space available in the cluster.
-- **USED:** The amount of raw storage consumed by user data.
+- **USED:** the space (accumulated over all OSDs) allocated for data (objects) keeping at block(slow) device. 
```

Comment:
the original is clearer to the user - you could add the extra clarification about the slow device by with something like:

The amount of raw storage consumed by user data (excluding BlueStore's db)

Reply:
Got it. I've updated it.